tests/fixtures/setup_database_fixture.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from typing import Tuple
+from exasol_transformers_extension.deployment.scripts_deployer import \
+    ScriptsDeployer
+from tests.utils.parameters import db_params, bucketfs_params
+
+
+bucketfs_connection_name = "TEST_TE_BFS_CONNECTION"
+schema_name = "TEST_INTEGRATION"
+language_alias = "PYTHON3_TE"
+
+
+def _create_schema(db_conn) -> None:
+    db_conn.execute(f"DROP SCHEMA IF EXISTS {schema_name} CASCADE;")
+    db_conn.execute(f"CREATE SCHEMA IF NOT EXISTS {schema_name};")
+
+
+def _deploy_scripts() -> None:
+    ScriptsDeployer.run(
+        dsn=db_params.address(),
+        user=db_params.user,
+        password=db_params.password,
+        schema=schema_name,
+        language_alias=language_alias
+    )
+
+
+def _create_bucketfs_connection(db_conn) -> None:
```

Comment:
I would maybe use the bucket_config and maybe use https://github.com/exasol/bucketfs-utils-python/blob/main/exasol_bucketfs_utils_python/bucketfs_utils.py#L107